vesting/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vesting"
+version = "0.1.0"
+authors = ["[] <[]>"]
```

Comment:
Please add authors like here https://github.com/paritytech/ink-examples/blob/3254a292b5cf13b087f92f2d02dc4bcbe847e71d/call-runtime/Cargo.toml#L4C1-L4C1
You can also leave yourself in the list

Reply:
done